README.md

```diff
@@ -642,31 +577,30 @@ Note that backslashes in double-quoted strings *must* be double-escaped and
 backslashes in single-quoted strings *may* be double-escaped.
 
 ##### `name`
-(**Namevar**: If ommitted, this attribute's value will default to the resource's
-title.)
 
-The package name. This is the name that the packaging system uses internally.
+Specifies the package name. This is the name that the packaging system uses internally. Valid options: String. Default: The resource's title.
```

Comment:
Hm, I guess that wasn't clear to me from the previous text. Based on the default, it doesn't seem like it's required for the user to set this parameter. From past modules, my understanding is that this is common behavior, and really it's the resource title that's required. Do they explicitly need to set `name => 'something'`? Or is it legal to not set `name` and allow it to use the resource title? Because from prior text it sounds like they don't have to set it as long as they're naming the resource (which they'd be doing anyway?)